ob_v2p1/schema/jsonld-context/context.json

```diff
@@ -0,0 +1,101 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+
+    "extensions": "https://w3id.org/openbadges/extensions#",
+    "obi": "https://w3id.org/openbadges#",
+    "validation": "obi:validation",
+
+    "cred": "https://w3id.org/credentials#",
+    "dc": "http://purl.org/dc/terms/",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "AlignmentObject": "schema:AlignmentObject",
+    "CryptographicKey": "sec:Key",
+    "Endorsement": "cred:Credential",
+
+    "Assertion": "obi:Assertion",
+    "BadgeClass": "obi:BadgeClass",
+    "Criteria": "obi:Criteria",
+    "Evidence": "obi:Evidence",
+    "Extension": "obi:Extension",
+    "FrameValidation": "obi:FrameValidation",
+    "IdentityObject": "obi:IdentityObject",
+    "Image": "obi:Image",
+    "HostedBadge": "obi:HostedBadge",
+    "hosted": "obi:HostedBadge",
```

Comment:
Should this be "Hosted" with a capital "H"?

Reply:
AFAIK, this is a legacy convention from the early days of the v0.5.0 & v1.0 spec (before the introduction of JSON-LD) where the type of verification methods was expressed as `hosted` or `signed`.

I don't advise introducing this breaking change in v2.1 but I would be on board with removing these aliases in v3.

Reply:
My mistake. I remembered the aliases, but not the case.